src/trip.js

```diff
@@ -20,6 +20,12 @@ class Trip {
     }, 0)
     return parseFloat(tripCost.toFixed(0));
   }
+
+  findTripName(destinations) {
+    let tripName = destinations.find(location => this.destinationID === location.id
+    )
```

Comment:
Nitpicky, but 26 can be on 25 unless you're doing stuff with it

Reply:
I love edits like this. Gonna nuke this whole method tho.